thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-wizard.component.tsx

```diff
@@ -230,7 +178,9 @@ export const SubscriptionGroupWizard: FunctionComponent<SubscriptionGroupWizardP
                                     disabled={!isSubscriptionGroupValid}
                                     onClick={handleSubmitClick}
                                 >
-                                    {submitBtnLabel}
+                                    {submitBtnLabel || isExisting
+                                        ? t("label.save")
+                                        : t("label.create")}
                                 </Button>
                             </Grid>
```

Comment:
Refactor to use `WizardBottomBar` 

https://github.com/startreedata/thirdeye/search?q=WizardBottomBar

Reply:
Implemented `WizardBottomBar`